modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -645,6 +645,17 @@ public Vector2(Vector2 v)
             return left;
         }
 
+        /// <summary>
+        /// Returns a Vector2 transformed (multiplied) by transformation matrix.
+        /// </summary>
+        /// <param name="t">transformation</param>
+        /// <param name="v">A Vector2 to transform.</param>
+        /// <returns>The transformed Vector2.</returns>
+        public static Vector2 operator *(Transform2D t, Vector2 v)
+        {
+            return v * t;
```

Comment:
This is not correct. Matrix multiplication is not commutative. `transform * vector` is the same as `transform.xform(vector)`, and `vector * transform` is the same as `transform.xform_inv(vector)`. See these lines in `core/variant/variant_op.cpp` for reference:

```cpp
register_op<OperatorEvaluatorXForm<Vector2, Transform2D, Vector2>>(Variant::OP_MULTIPLY, Variant::TRANSFORM2D, Variant::VECTOR2);
register_op<OperatorEvaluatorXFormInv<Vector2, Vector2, Transform2D>>(Variant::OP_MULTIPLY, Variant::VECTOR2, Variant::TRANSFORM2D);
```

Reply:
I don't have enough knowledge. I'm sorry

I'll know where to look next time there are similar changes